### ISSUE TITLE
Error when G55 is selected and GCode is run

### ISSUE THREAD

**Issue by fschramek** [kcjengr/qtpyvcp#149]
Hi, 
i get this error when trying to open Gcode files or subroutines through the probe basic screen, while having enabled the G55 Work offset. Can i only open Programs with G55 in them, and a G54 will break the system, or do i have setup something wrong?

Error:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/qtpyvcp/widgets/display_widgets/vtk_backplot/vtk_backplot.py", line 825, in load_program
    point_01_pos = self.path_offset_start_point[prev_wcs_index]
                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^
KeyError: 0

After this, i have to restart the machine.

Has someone else had this problem?
Thanks in advance
Felix

**Comment by Lcvette** [comment]
Yes, this is currently a bug in the develop version. it is not present in the stable version.  I experienced it yesterday also and we are actively working on the solution, it is something broken in the vtk backplotter.  you can change to the stable version if you would like to run without the error until we get the bug fixed in the develop version.  please see the docs on changing from one o the other here:

https://kcjengr.github.io/probe_basic/stable_develop_branch_change.html
